remote-wiring-experience/ConnectionPage.xaml

```diff
@@ -86,6 +87,7 @@
                                     <x:String>Bluetooth</x:String>
                                     <x:String>USB</x:String>
                                     <x:String>Network</x:String>
+                                    <x:String>BLE</x:String>
```

Comment:
This list seems jumbled. How about alphabetical order. Although, this goes directly against my other observation, can we put USB as default because it is the most common work out of the box solution.